website/blog/2020-01-10-cobalt.md

```diff
@@ -0,0 +1,718 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.0 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+After a long while of silence and with loads of new features we are excited to
+announce the release of Metals v0.8.0 codename `Cobalt`. This release includes a
+huge number of new features and improvements.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+along with contributors from the community.
+
+In this release we merged 201 PRs and closed 24 issues, full details:
+https://github.com/scalameta/metals/milestone/9?closed=1
+
+## TL;DR
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+- goto implementation
+- rename symbol
+- support for Pants build tool
+- run and debug for tests and main methods
+- breakpoints
+- improvements for Bloop integration
+- first code action - import missing symbol
+- worksheet support
+- Add completion to add all abstract members To mention
+
+## Goto implementation
+
+TODO
+
+## Rename symbol
+
+TODO
+
+## Support for Pants build tool
+
+TODO
+
+## Run and debug for tests and main methods
+
+TODO @marek1840 ?
+
+## Breakpoints
+
+TODO @marek1840 ?
+
+## Improvements for Bloop integration
+
+TODO
+
+Restart Bloop automatically if newer version is needed Automatically reconnct
+Switch to to using Bloop Launcher Remove separate plugin for Metals Bloop itself
+resolves semanticdb
+
+## Support for java 11
+
+TODO
+
+Currently tested on CI No warning when using 11 and it will be detected
+automatically
+
+## Worksheet support
+
+TODO
+
+via decoration protocol for coc-metals and vs code via applyEdit and comments +
```

Comment:
Decoration protocol for `coc-metals` isn't finished yet. Depending on when we will actually release, it may be, but as of now it's not.

Reply:
Actually, keep it in. Just merged it in this morning.

Reply:
Cool! :D